posthog/api/property_definition.py

```diff
@@ -22,6 +23,32 @@ def update(self, property_definition: PropertyDefinition, validated_data):
         raise EnterpriseFeatureException()
 
 
+def is_pg_trgm_installed():
+    try:
+        with connection.cursor() as cursor:
+            cursor.execute("SELECT COUNT(*) FROM pg_extension WHERE extname = 'pg_trgm'")
+            row = cursor.fetchone()
+            has_extension = bool(row) and bool(row[0])
+        with connection.cursor() as cursor:
+            cursor.execute(
+                """
+                SELECT
+                    i.relname as index_name
+                FROM
+                    pg_class i,
+                    pg_index ix
+                WHERE
+                    i.oid = ix.indexrelid
+                    and i.relname = 'index_property_definition_name';
+            """
+            )
+            row = cursor.fetchone()
+            has_index = bool(row) and bool(row[0])
+        return has_extension and has_index
+    except BaseException:
```

Comment:
Using BaseException here (and everywhere else) seems not well-justified and like a bad idea. 

Do you know what the error is when we want handle the failure like this?